src/diffusers/__init__.py

```diff
@@ -130,6 +131,7 @@
         StableDiffusionUpscalePipeline,
         StableUnCLIPImg2ImgPipeline,
         StableUnCLIPPipeline,
+        TextToVideoMSPipeline,
```

Comment:
Should this be just `TextToVideoPipeline`?

Reply:
Changed to TextToVideoSDPipeline SD for stable diffusion